flipperdevices/flipperzero-firmware
# 125 kHz/LFRFID - New Feature - Direct Emulation

# What's new

- **New Menu item for the 125 kHz/LFRFID application - "Emulate Directly"**
- Allows you to select a protocol, enter the hex digits, and emulate immediately - bypassing the need to load/save files

Created some new scenes in applications/lfrfid/scenes:
- lfrfid_app_scene_emu_data
- lfrfid_app_scene_emu_menu
- Also updated some core files to enumerate the new scenes (i messed up the diff so i'm very sorry about that, the only updates I made were in line with the established pattern and do not touch any logic)

# Verification 

- Recompile the firmware and load it on ur flipper
- Check "125 kHz RFID" application for a 4th menu item - **Emulate Directly**
- Select protocol
- Enter hex digits and hit "save"
- Flippy should now be emulating the digits u entered
- Hitting back will take u back to byte edit screen

# Checklist (For Reviewer)
